src/sentry/tasks/store.py

```diff
@@ -625,6 +627,23 @@ def _do_save_event(
                     },
                 )
 
+                metrics.timing(
+                    "events.start-to-start-save-event",
+                    start_save_event_time - start_time,
+                    instance=data["platform"],
+                )
+
+            metrics.timing(
+                "events.start-save-event-to-process",
```

Comment:
```suggestion
                "events.start_save_event_to_processed",
```
"process" -> "processed" since processing is complete now